Filepath file
---
bool success
string error_msg
Multihash ipfs_address